### ISSUE TITLE
Quicker way to release GAX and google auth library together?

### ISSUE THREAD

**Issue by suztomo** [GoogleCloudPlatform/cloud-opensource-java#2137]
As of now, when the google auth library provides a new method for GAX, we need to release Google Cloud BOM and the Libraries BOM that contains the newer version of the google auth library. Until that, GAX cannot use the google auth library because Linkage Monitor complains:

https://github.com/googleapis/gax-java/pull/1420#issuecomment-884371336

Once the Libraries BOM release is done, we can upgrade the google auth library version in GAX.

Is there a better way?

**Comment by elharo** [comment]
How often does this happen? If it's a once a year thing, we might as well just push libraries-bom.

This is also an argument for not necessarily waiting for google-cloud-bom to update the auth-library before libraries-bom does.

**Comment by suztomo** [comment]
Yes, it's rare. For this case, Neenu is going to release the Google Cloud BOM (with new dependencies BOM), and we'll release the Libraries BOM.

I'll record occurrences in this issue in case similar situation happens again.

**Comment by suztomo** [comment]
Also I'm wondering whether Linkage Monitor is a suitable tool here now. The Libraries BOM now follows the Google Cloud BOM and the shared dependencies BOM.

While Linkage Monitor might find some unexpected dependency problems, the main purpose of finding incompatible dependencies are handled by the full convergence check at java-cloud-bom repository.

(I have not decided anything yet.)
